build-images/src/Dockerfile

```diff
@@ -101,6 +102,12 @@ RUN curl -L https://foundry.paradigm.xyz | bash \
 # It also means we're effectively testing the devbox in CI.
 # We use this image for test isolation.
 FROM base-build AS build
+
+# APT repo for node (22.15.0)
+RUN curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+ENV NODE_MAJOR=22
+RUN echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_$NODE_MAJOR.x nodistro main" | tee /etc/apt/sources.list.d/nodesource.list
```

Comment:
does .x here mean latest against this major, should we not be pinned to a specific release

Reply:
Previously we were pinned to whatever ubuntu:noble shipped, which was 18.X.X...I think as long as we stick to the major, it should be fine.